connect/connect.go

```diff
@@ -1,24 +1,61 @@
 package connect
 
 import (
+	"errors"
 	"fmt"
+	"os"
+	"path/filepath"
 
 	"github.com/joshmedeski/sesh/config"
-	"github.com/joshmedeski/sesh/session"
 	"github.com/joshmedeski/sesh/tmux"
 	"github.com/joshmedeski/sesh/zoxide"
 )
 
-func Connect(choice string, alwaysSwitch bool, command string, config *config.Config) error {
-	session, err := session.Determine(choice, config)
+func Connect(
+	choice string,
+	alwaysSwitch bool,
+	command string,
+	config *config.Config,
+) error {
+	var errorStack []error
```

Comment:
Can you explain why you moved away from the detemrine logic here? I preferred that logic being separated and I haven't seen the errStack pattern before.